quorum/start-node.sh

```diff
@@ -69,7 +69,7 @@ elif [ "$SERVICE" = "geth" ]; then
     /usr/local/bin/geth --datadir /qdata/dd init /qdata/genesis.json
   fi
 
-  GETH_ARGS="--datadir /qdata/dd --raft --rpc --rpcaddr 0.0.0.0 --port "$ETH_PORT" --ws --wsorigins="*" --wsaddr 0.0.0.0 --wsport "$WEBSOCKET_PORT" --rpcport "$RPC_PORT" --raftport "$RAFT_PORT" --wsapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum --rpcapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum --nodiscover --unlock 0 --password /qdata/passwords.txt"
+  GETH_ARGS="--datadir /qdata/dd --raft --raftjoinexisting 2 --rpc --rpcaddr 0.0.0.0 --port "$ETH_PORT" --ws --wsorigins="*" --wsaddr 0.0.0.0 --wsport "$WEBSOCKET_PORT" --rpcport "$RPC_PORT" --raftport "$RAFT_PORT" --wsapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,raft --rpcapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,raft --nodiscover --unlock 0 --password /qdata/passwords.txt"
```

Comment:
This is the command that starts Geth in the processor. This file goes inside the processors Docker image